Sources/MixpanelInstance.swift

```diff
@@ -37,6 +37,8 @@ public protocol MixpanelDelegate: AnyObject {
 
 public typealias Properties = [String: MixpanelType]
 typealias InternalProperties = [String: Any]
+typealias TimedEventID = String
+typealias TimedEvents = [TimedEventID: TimeInterval]
```

Comment:
I added these two typealiases for additional type-safely and clarity. Ideally TimedEventID is it's own type that wraps a string but I didn't want to change the call site too drastically from what it already is.